src/Services/ShipmentNotificationService.php

```diff
@@ -368,7 +369,8 @@ public function prepareRequestDto(Order $order): RequestDTO
       $products = $purchaseOrderInfo['products'];
 
       //Decide how to get tracking and package information.
-      if ($this->shipmentProviderService->isShippingWithWayfair() || empty($plentymarketsShippingInformation->shippingServiceProvider)) {
+      // if ($this->shipmentProviderService->isShippingWithWayfair()) {
+      if ($this->shipmentProviderService->isShippingWithWayfair()){ //|| empty($plentymarketsShippingInformation->shippingServiceProvider)) {
```

Comment:
Where/how do we handle the situation of no `shippingServiceProvider` after these changes?